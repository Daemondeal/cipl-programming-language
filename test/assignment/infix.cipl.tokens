Token          Lexeme         Line 
LET            let            1    
IDENTIFIER     a              1    
EQUAL          =              1    
STRING         "a"            1     a
LINE_END       \n             2    
LET            let            2    
IDENTIFIER     b              2    
EQUAL          =              2    
STRING         "b"            2     b
LINE_END       \n             3    
IDENTIFIER     a              3    
PLUS           +              3    
IDENTIFIER     b              3    
EQUAL          =              3    
STRING         "value"        3     value
LINE_END       \n             3    
EOF                           3    
