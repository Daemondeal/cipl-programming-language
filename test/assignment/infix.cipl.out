::stdout::

::stderr::
Error on line 3 at `=`: Invalid assignment target.
